src/projects/schemas/project.schema.ts

```diff
@@ -0,0 +1,9 @@
+import * as mongoose from 'mongoose';
+
+export const ProjectSchema = new mongoose.Schema({
+      _id: Number,
+      name: String,
+      description: String,
+      url: String,
+      isLookingForContributors: Boolean
+    }, { _id: false });
```

Comment:
Here we need add field creator and contributors because now we don't have information who is creator and who is contributors